README.md

```diff
@@ -42,10 +42,10 @@ import DropZone from '@4tw/vue-drop-zone'
 ### Properties
 
 - endpoint[String:document.URL]: Describes the endpoint where the dropzone should upload the files to
-- mode[String:'TUS']: Describes the uploader to be used. Can either be 'TUS' or 'XHR'.
+- uploader[Object:{ uploader:Class, options:Object }]: Describes the uploader to be used. Uppy already provides two uploaders for XHR: https://uppy.io/docs/xhr-upload/#Options or TUS: https://uppy.io/docs/tus/#Options.
 - options[Object:{}]:
   - uppy[Object:{}] for the uppy client options (https://uppy.io/docs/uppy/#Options),
-  - uploader[Object:{}] for the uploader options (XHR: https://uppy.io/docs/xhr-upload/#Options), (TUS: https://uppy.io/docs/tus/#Options)
+  - uploader[Object:{}] for the uploader options
```

Comment:
I'd remove that since the options for the uploader can already be passed using the `uploader` property.